.github/ISSUE_TEMPLATE/Sample_issue.md

```diff
@@ -1,12 +1,12 @@
 ---
-name: Problem with a policy
-about: If you have a problem, bug, or enhancement with a policy.
+name: Problem with a policy sample
+about: If you discover a problem, bug, or enhancement with a policy sample in this repository.
 ---
 <!--
-    Your feedback and support is greatly appreciated, thanks for contributing!
+    Your feedback and support of these samples is greatly appreciated, thanks for contributing!
```

Comment:
>Your feedback and support of these samples is greatly appreciated, thanks for contributing! [](start = 4, length = 91)

I think your blurb about Support from the README could go here too, since most people will open an issue without reading the README

Reply:
Good idea, adding.